META.github.template

```diff
@@ -0,0 +1,11 @@
+# JBUILDER_GEN
+
+package "js" (
+  description = "Deprecated. Use github-js directly"
+  requires = "cohttp-js"
+)
+
+package "unix" (
+  description = "Deprecated. Use github-unix directly"
+  requires = "github-unix"
+)
```

Comment:
The problem is that depending on `github-unix` will implicitly depend on the next release and this release must exist for packages to depend on. We've got some other deprecated cruft so it seems simpler to just break compat for the small number of downstream packages.